Build fails

Build at commit 1d6d14e6f710b74b08291e8e94c7810d3f2ccff9 fails with `Could not find build ':kotlin-plugins:generatePrecompiledScriptPluginAccessors:accessors8052664764592233112'`

```
michel@debian:~/git/Android-Password-Store$ ./gradlew assembleFreeDebug
Downloading https://services.gradle.org/distributions/gradle-7.4.2-bin.zip
...........10%...........20%...........30%...........40%...........50%...........60%...........70%...........80%...........90%...........100%

Welcome to Gradle 7.4.2!

Here are the highlights of this release:
 - Aggregated test and JaCoCo reports
 - Marking additional test source directories as tests in IntelliJ
 - Support for Adoptium JDKs in Java toolchains

For more details see https://docs.gradle.org/7.4.2/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
Configuration cache is an incubating feature.
Calculating task graph as no configuration cache is available for tasks: assembleFreeDebug
Type-safe project accessors is an incubating feature.

> Task :build-logic:android-plugins:compileKotlin
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /home/michel/.gradle/wrapper/dists/gradle-7.4.2-bin/48ivgl02cpt2ed3fh9dbalvx8/gradle-7.4.2/lib/kotlin-stdlib-1.5.31.jar (version 1.5)
    /home/michel/.gradle/wrapper/dists/gradle-7.4.2-bin/48ivgl02cpt2ed3fh9dbalvx8/gradle-7.4.2/lib/kotlin-stdlib-common-1.5.31.jar (version 1.5)
    /home/michel/.gradle/wrapper/dists/gradle-7.4.2-bin/48ivgl02cpt2ed3fh9dbalvx8/gradle-7.4.2/lib/kotlin-stdlib-jdk7-1.5.31.jar (version 1.5)
    /home/michel/.gradle/wrapper/dists/gradle-7.4.2-bin/48ivgl02cpt2ed3fh9dbalvx8/gradle-7.4.2/lib/kotlin-stdlib-jdk8-1.5.31.jar (version 1.5)
    /home/michel/.gradle/wrapper/dists/gradle-7.4.2-bin/48ivgl02cpt2ed3fh9dbalvx8/gradle-7.4.2/lib/kotlin-reflect-1.5.31.jar (version 1.5)
    /home/michel/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.6.10/e80fe6ac3c3573a80305f5ec43f86b829e8ab53d/kotlin-stdlib-jdk8-1.6.10.jar (version 1.6)
    /home/michel/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.5.31/1523fcd842a47da0820cea772b19c51056fec8a9/kotlin-reflect-1.5.31.jar (version 1.5)
    /home/michel/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.6.10/e1c380673654a089c4f0c9f83d0ddfdc1efdb498/kotlin-stdlib-jdk7-1.6.10.jar (version 1.6)
    /home/michel/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.6.10/b8af3fe6f1ca88526914929add63cf5e7c5049af/kotlin-stdlib-1.6.10.jar (version 1.6)
    /home/michel/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.6.10/c118700e3a33c8a0d9adc920e9dec0831171925/kotlin-stdlib-common-1.6.10.jar (version 1.6)
w: Consider providing an explicit dependency on kotlin-reflect 1.6 to prevent strange errors
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath

0 problems were found storing the configuration cache.

See the complete report at file:///home/michel/git/Android-Password-Store/build/reports/configuration-cache/d0vqg3qsz9vz6tbdl5uqr933q/607dtob64f05qmsyovxit6yfa/configuration-cache-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not find build ':kotlin-plugins:generatePrecompiledScriptPluginAccessors:accessors8052664764592233112'

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 41s
30 actionable tasks: 30 executed
Configuration cache entry stored.
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -51,6 +51,17 @@ The app comes in two 'flavors', a FOSS-only **free** variant and a **nonFree** v
 
 You can find the generated APK at `app/outputs`.
 
+The project makes use of the unstable [Configuration Cache](https://docs.gradle.org/current/userguide/configuration_cache.html) feature of Gradle. This has the potential to cause spurious build failures that look something like this:
+
+```
+FAILURE: Build failed with an exception.
+
+* What went wrong:
+Could not find build ':kotlin-plugins:generatePrecompiledScriptPluginAccessors:accessors8052664764592233112'
+```
+
+This can be resolved by simply re-running the command, or adding the `--no-configuration-cache` parameter.
+
 ## Pre-push checks
 
 The project enforces codestyle conventions and library API stability by virtue of a carefully curated Gradle build. To setup a Git pre-push hook to run them automatically, run `./gradlew installGitHooks`.

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -21,7 +21,7 @@
     android:required="false" />
 
   <application
-    android:name=".Application"
+    android:name="dev.msfjarvis.aps.Application"
     android:allowBackup="false"
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
@@ -31,23 +31,23 @@
     tools:ignore="GoogleAppIndexingWarning">
 
     <activity
-      android:name=".ui.passwords.PasswordStore"
+      android:name="dev.msfjarvis.aps.ui.passwords.PasswordStore"
       android:configChanges="orientation|screenSize" />
 
     <activity
-      android:name=".ui.onboarding.activity.OnboardingActivity"
+      android:name="dev.msfjarvis.aps.ui.onboarding.activity.OnboardingActivity"
       android:configChanges="orientation|screenSize" />
 
     <activity
-      android:name=".ui.proxy.ProxySelectorActivity"
+      android:name="dev.msfjarvis.aps.ui.proxy.ProxySelectorActivity"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.crypto.DecryptActivityV2"
+      android:name="dev.msfjarvis.aps.ui.crypto.DecryptActivityV2"
       android:exported="true" />
 
     <activity
-      android:name=".ui.main.LaunchActivity"
+      android:name="dev.msfjarvis.aps.ui.main.LaunchActivity"
       android:configChanges="orientation|screenSize"
       android:exported="true"
       android:theme="@style/NoBackgroundThemeM3">
@@ -66,54 +66,54 @@
       tools:node="replace" />
 
     <activity
-      android:name=".ui.git.config.GitServerConfigActivity"
+      android:name="dev.msfjarvis.aps.ui.git.config.GitServerConfigActivity"
       android:label="@string/title_activity_git_clone"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.git.config.GitConfigActivity"
+      android:name="dev.msfjarvis.aps.ui.git.config.GitConfigActivity"
       android:label="@string/title_activity_git_config"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.git.log.GitLogActivity"
+      android:name="dev.msfjarvis.aps.ui.git.log.GitLogActivity"
       android:label="@string/title_activity_git_log" />
 
     <activity
-      android:name=".ui.settings.SettingsActivity"
+      android:name="dev.msfjarvis.aps.ui.settings.SettingsActivity"
       android:label="@string/action_settings"
       android:parentActivityName=".ui.passwords.PasswordStore" />
 
     <activity
-      android:name=".ui.settings.DirectorySelectionActivity"
+      android:name="dev.msfjarvis.aps.ui.settings.DirectorySelectionActivity"
       android:theme="@style/NoBackgroundThemeM3" />
 
     <activity
-      android:name=".ui.crypto.PasswordCreationActivity"
+      android:name="dev.msfjarvis.aps.ui.crypto.PasswordCreationActivity"
       android:label="@string/new_password_title"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.crypto.PasswordCreationActivityV2"
+      android:name="dev.msfjarvis.aps.ui.crypto.PasswordCreationActivityV2"
       android:label="@string/new_password_title"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.crypto.DecryptActivity"
+      android:name="dev.msfjarvis.aps.ui.crypto.DecryptActivity"
       android:windowSoftInputMode="adjustResize" />
 
     <activity
-      android:name=".ui.crypto.GetKeyIdsActivity"
+      android:name="dev.msfjarvis.aps.ui.crypto.GetKeyIdsActivity"
       android:theme="@style/NoBackgroundThemeM3" />
 
     <service
-      android:name=".util.services.ClipboardService"
+      android:name="dev.msfjarvis.aps.util.services.ClipboardService"
       android:process=":clipboard_service_process" />
     <service
-      android:name=".util.services.PasswordExportService"
+      android:name="dev.msfjarvis.aps.util.services.PasswordExportService"
       android:process=":password_export_service_process" />
     <service
-      android:name=".util.services.OreoAutofillService"
+      android:name="dev.msfjarvis.aps.util.services.OreoAutofillService"
       android:exported="true"
       android:permission="android.permission.BIND_AUTOFILL_SERVICE">
       <intent-filter>
@@ -124,40 +124,40 @@
         android:resource="@xml/oreo_autofill_service" />
     </service>
 
-    <activity android:name=".ui.folderselect.SelectFolderActivity" />
+    <activity android:name="dev.msfjarvis.aps.ui.folderselect.SelectFolderActivity" />
     <activity
-      android:name=".ui.sshkeygen.SshKeyImportActivity"
+      android:name="dev.msfjarvis.aps.ui.sshkeygen.SshKeyImportActivity"
       android:theme="@style/NoBackgroundThemeM3"
       android:windowSoftInputMode="adjustResize" />
     <activity
-      android:name=".ui.sshkeygen.SshKeyGenActivity"
+      android:name="dev.msfjarvis.aps.ui.sshkeygen.SshKeyGenActivity"
       android:label="@string/pref_ssh_keygen_title"
       android:windowSoftInputMode="adjustResize" />
     <activity
-      android:name=".ui.autofill.AutofillDecryptActivity"
+      android:name="dev.msfjarvis.aps.ui.autofill.AutofillDecryptActivity"
       android:theme="@style/NoBackgroundThemeM3" />
     <activity
-      android:name=".ui.autofill.AutofillDecryptActivityV2"
+      android:name="dev.msfjarvis.aps.ui.autofill.AutofillDecryptActivityV2"
       android:theme="@style/NoBackgroundThemeM3" />
     <activity
-      android:name=".ui.autofill.AutofillFilterView"
+      android:name="dev.msfjarvis.aps.ui.autofill.AutofillFilterView"
       android:configChanges="orientation|keyboardHidden"
       android:theme="@style/DialogLikeThemeM3"
       android:windowSoftInputMode="adjustNothing" />
     <activity
-      android:name=".ui.autofill.AutofillSaveActivity"
+      android:name="dev.msfjarvis.aps.ui.autofill.AutofillSaveActivity"
       android:theme="@style/NoBackgroundThemeM3" />
     <activity
-      android:name=".autofill.oreo.ui.AutofillSmsActivity"
+      android:name="dev.msfjarvis.aps.autofill.oreo.ui.AutofillSmsActivity"
       android:configChanges="orientation"
       android:theme="@style/DialogLikeThemeM3"
       android:windowSoftInputMode="adjustNothing" />
     <activity
-      android:name=".ui.autofill.AutofillPublisherChangedActivity"
+      android:name="dev.msfjarvis.aps.ui.autofill.AutofillPublisherChangedActivity"
       android:configChanges="orientation|keyboardHidden"
       android:theme="@style/DialogLikeThemeM3"
       android:windowSoftInputMode="adjustNothing" />
-    <activity android:name=".ui.pgp.PGPKeyImportActivity"
+    <activity android:name="dev.msfjarvis.aps.ui.pgp.PGPKeyImportActivity"
       android:theme="@style/NoBackgroundThemeM3" />
   </application>
 

